.changeset/dirty-bobcats-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-cloudflare": patch
+---
+
+Fixes c3 usage with yarn
```

Comment:
Can we have a bit more explanation here? This will appear in the changelog for the package. We should write from the point of view of someone trying to find out what has changed in a release.